modules.json

```diff
@@ -4,139 +4,157 @@
     "repos": {
```

Comment:
You will love this one :eyes: , there is a new version of all these modules after this PR has been merged https://github.com/nf-core/modules/pull/1261

Reply:
Bah, it can wait for the next release now that we have tested everything. Didn't even see that it was merged! Thought we were still discussing it 👀